### DISCUSSION TITLE
I encountered an issue when trying to open the URL.

### DISCUSSION THREAD

**Discussion by kenlinkenlin21** [open-webui/open-webui#1682] [General]
**Bug Summary:**
My open-webui website shows nothing but a white screen.

**Expected Behavior:**
Hi, I am a beginner with open-webui. Today, I tried installing it without Docker. After following all the steps, I ran the 'start_windows.bat'. It appeared to be successful.
![image](https://github.com/open-webui/open-webui/assets/47999979/63c43d3f-6122-4859-a82e-c49ef573ac87)

However, the website shows nothing but a white screen 
![image](https://github.com/open-webui/open-webui/assets/47999979/d72265f7-9abf-40ed-aeab-b1da77e14792)
 Then I tried to check the console of chrome
![image](https://github.com/open-webui/open-webui/assets/47999979/2da87042-a31b-4267-82de-d8b095856200)

If someone encounters the same issue, they can provide suggestions, and I would be grateful.

## Environment

- **Open WebUI Version:**  0.1.120
- **Ollama (if applicable):** 0.1.8

- **Operating System:**  Windows 11
- **Browser (if applicable):** Chrome  123.0.6312.124

**Comment by justinh-rahb** [comment]
Did you follow all of the instructions? The frontend must be built first, it's not just a matter of running `start_windows.bat`, that's only the last step:

```bash
cd open-webui
cp .env.example .env
npm i
npm run build
cd backend
pip3 install -r requirements.txt -U
start_windows.bat
```

**Reply by kenlinkenlin21** [reply]
Yes, I have followed all the steps.
However, windows system can't use cp command. I use 'copy .env.example .env' instead of 'cp .env.example .env'
Could this be the reason?

**Reply by justinh-rahb** [reply]
Wouldn't matter how, just as long as it's the same contents as the example.

**Reply by kenlinkenlin21** [reply]
Hmm... i tried these steps in two computer and got same problem. 
Thank you for your response, perhaps I should take another look to see if there are any other issues elsewhere.

**Comment by justinh-rahb** [comment]
> * Ollama (if applicable): 0.1.8

OH I didn't see this before... is this a typo? That version is _ancient_ and could go well towards explaining the behaviour here.

**Reply by kenlinkenlin21** [reply]
Oh... it's my fault. I wrote the version of the ollama-python package.
The version of ollama on my computer is 0.1.31.

**Reply by justinh-rahb** [reply]
Okay ya that version is fine then. Could you possibly try another older version of WebUI?
```bash
git checkout v0.1.119
```
And repeat the installation steps before starting again.

**Reply by kenlinkenlin21** [reply]
I tried the v0.1.119
Unfortunately, the problem persists. I also test in Edge.
There are some screenshots of the logs.
![image](https://github.com/open-webui/open-webui/assets/47999979/86d814a4-59ae-445a-a184-625d4aa0114b)

On Chrome
![image](https://github.com/open-webui/open-webui/assets/47999979/1e00391a-4897-4f73-8c08-1f951868047b)

On Edge
![image](https://github.com/open-webui/open-webui/assets/47999979/c5789ea8-8233-498f-95a1-bcf3f90e3adc)

**Comment by WHG555** [comment]
I'm having the same problem, can you fix it?

**Reply by kenlinkenlin21** [reply]
Unfortunately, no.
